plotting/timeseries.py

```diff
@@ -72,36 +71,34 @@ def load_data(self):
             else:
                 self.depth_label = ''
 
-            if not (set(dataset.variables[var].dimensions) &
+            # Override depth request if requested variable has no depth (i.e. surface only)
+            if not (set(dataset.variables[variable].dimensions) &
                     set(dataset.nc_data.depth_dimensions)):
                 self.depth = 0
 
-            times = None
             point_data = []
+            dep = []
             for p in self.points:
                 data = []
-                for v in self.variables:
```

Comment:
No need for a loop here since we have the calculation layer to produce correct magnitude values. This plot doesn't support multiple variables on the same plot anyway.